roles/common/tasks/monitoring.yml

```diff
@@ -225,3 +225,7 @@
     check: "{{ sensu_checks.common.check_raid }}"
   notify: restart sensu-client missing ok
   when: common.hwraid.enabled|bool
+
+- name: setup vyatta snmp checks on controllers
```

Comment:
Only thing wrong here is that includes do not get a name.

Instead of having all of this in a file with a conditional attached, we could just do it all as a block in the monitoring.yml file. http://docs.ansible.com/ansible/playbooks_blocks.html